hoot-core/src/main/cpp/hoot/core/io/DataConverter.cpp

```diff
@@ -270,7 +270,7 @@ void DataConverter::_convertToOgr(const QStringList& inputs, const QString& outp
       numTasks++;
     }
     int currentTask = 1;
-    const float taskWeight = 1.0 / (float)numTasks;
+    double taskWeight = 1.0 / numTasks;
```

Comment:
Revert this and all other calculations of `taskWeight` in this file because it should be a `float`.